static/app/stores/organizationStore.tsx

```diff
@@ -24,22 +20,14 @@ interface OrganizationStoreDefinition extends CommonStoreDefinition<State> {
   get(): State;
   init(): void;
   onFetchOrgError(err: RequestError): void;
-  onUpdate(org: Organization, options?: UpdateOptions): void;
+  onUpdate(org: Organization, options?: {replace: true}): void;
```

Comment:
two `onUpdate`? 👀

Reply:
Yeah this is an override so when `replace` is true the full organization is required